helm-charts/flink-historyserver/templates/efs.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: efs-flink-history
+parameters:
+  provisioningMode: efs-ap
+  fileSystemId: "{{- .Values.aws_efs_file_system_id }}"
+provisioner: "efs.csi.aws.com"
+---
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: flink-historyserver-efs-pv
+spec:
+  capacity:
+    storage: "1Mi"
+  volumeMode: "Filesystem"
+  accessModes:
+    - "ReadWriteMany"
+  persistentVolumeReclaimPolicy: Retain
```

Comment:
This means that the EFS volumes must be manually cleaned up when testing is done. I think that's fine, but can you add a comment here about that?

Reply:
left the comment